cluster/saltbase/salt/kube-node-unpacker/kube-node-unpacker.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Congrats, you're shellcheck clean. ;) (Maybe you caught an earlier email from me.)